subprojects/gradle-guides-plugin/gradle/functional-test.gradle

```diff
@@ -17,8 +17,8 @@ task functionalTest(type: Test) {
     systemProperty("gradle.version", gradle.gradleVersion)
 
     reports {
-        html.destination = project.file("$html.destination/functional")
-        junitXml.destination = project.file("$junitXml.destination/functional")
+        html.outputLocation = project.file("${html.outputLocation.get().asFile}/functional")
+        junitXml.outputLocation = project.file("${junitXml.outputLocation.get().asFile}/functional")
     }
```

Comment:
This is the default, so I think we can just delete this.

Reply:
Ah right, now it is `functionalTest/functional`, but `functionalTest` should be good enough.